src/models/dbModels.go

```diff
@@ -34,6 +23,11 @@ func (Users) TableName() string {
 	return constants.UserTable
 }
 
+type DeleteWatchlistScripsRequest struct {
+	WatchlistName string `gorm:"column:watchlist_name" json:"watchlistName"`
+	Scrips        []int  `gorm:"column:symbol" json:"scrips"`
+}
```

Comment:
Why is this model required?  make the changes as required

Reply:
Resolved